samples/java-protobuf-customer-registry-kafka-quickstart/docker-compose.yml

```diff
@@ -3,11 +3,15 @@ services:
   kalix-runtime:
```

Comment:
Once we agree that the docker-compose change is ok, I will change all compose files in samples.

Reply:
unless in a specific sample we demonstrate something that needs persistence, I would not change all the sample compose files to include this. I think maybe in the archetype we can put this (we have other flags there as well) but on the samples I don't see much benefit.

Reply:
I'm a bit biased because I was using this a lot the last few days, but for me, the benefit is that the config is there, I don't have to look for it in the docs and copy-paste some lines. It's just a matter of uncommenting and I'm ready to go.